tox.ini

```diff
@@ -34,7 +34,7 @@ commands=
     native-blockchain-constantinople: pytest {posargs:tests/json-fixtures/blockchain/test_blockchain.py --fork Constantinople}
     native-blockchain-petersburg: pytest {posargs:tests/json-fixtures/blockchain/test_blockchain.py --fork ConstantinopleFix}
     native-blockchain-istanbul: pytest {posargs:tests/json-fixtures/blockchain/test_blockchain.py --fork Istanbul}
-    native-blockchain-berlin: pytest {posargs:tests/json-fixtures/blockchain/test_blockchain.py -k "not modexp" --fork Berlin}
```

Comment:
:tada: Won't be able to look at the code till tomorrow, but it's great that all tests pass now! :D